src/core/component.test.ts

```diff
@@ -744,6 +744,28 @@ describe("nested option", () => {
         });
     });
 
+    it("component shouldn't set option if option already set (v-model)", (done) => {
+        new Vue({
+            template:
+                `<test-component v-model="value">` +
```

Comment:
Is there any difference in comparison with  `<test-component prop1="value">`  ?

Reply:
it is reproduced only with two-way binding

Reply:
Ok